.vscode/schemas/define.schema.json

```diff
@@ -29,8 +29,7 @@
 						"flash",
 						"php",
 						"cpp",
-						"cs",
-						"java",
```

Comment:
It might be better to keep them around for editing older (haxe 4.x) files?

Reply:
Right, schemas still live on haxe repo.. but iirc we wanted to change that at some point, and it doesn't hurt